vip-cache-manager/js/admin-bar.js

```diff
@@ -0,0 +1,69 @@
+(async () => {
+	async function postData(url = '', data = {}) {
+		// Default options are marked with *
+		const response = await fetch(url, {
+			method: 'POST',
+			credentials: 'same-origin',
+			headers: {
+				'Content-Type': 'application/json'
+			},
+			body: JSON.stringify({ ...data })
+		});
+		return response.json();
+	}
+
+	// Indicates whether there's a purge request happening
+	let purgeQueried = false;
```

Comment:
Minor: This would be better named as `purgeInProgress`.

Reply:
Ha, that was supposed to be `purgeQueued` :)